roles/hosted_engine_setup/tasks/apply_openscap_profile.yml

```diff
@@ -22,8 +22,8 @@
   command: >-
     oscap xccdf eval --profile {{ oscap_profile.stdout }} --remediate
     --report /root/openscap-report.html {{ oscap_datastream }}
-  failed_when: false
-  ignore_errors: true
+  register: oscap_applied
+  failed_when: oscap_applied.rc == 1
```

Comment:
Actually we should differentiate between rc=1 and rc=2. 
According to oscap man page:
`oscap returns 0 if all rules pass. If there is an error during evaluation, the return code is 1. If there is at least one rule with either fail or unknown result, oscap-scan finishes with return code 2.`

Reply:
Indeed, AFAICT when applying security profile on HE vm (and also on RHVH via anconda) there are many rules that fail, this is why deployment will fail only when `rc == 1`.
I also opened https://bugzilla.redhat.com/show_bug.cgi?id=1983476.

Reply:
I added a comment